README.md

```diff
@@ -246,6 +246,37 @@ The command below can be used to run the glibc tests:
 
     make check-glibc-linux
 
+### LLVM / clang
+
+LLVM can be used in combination with the RISC-V GNU Compiler Toolchain
+to build RISC-V applications. To build LLVM with C and C++ support the
+configure flag `--enable-llvm` can be used.
+
+E.g. to build LLVM on top of a RV64 Linux toolchain the following commands
+can be used:
+
+  ./configure --prefix=$RISCV --enable-llvm --enable-linux
+  make
```

Comment:
This should presumably be wrapped in code tags as otherwise the two lines become one and thus an incorrect command line:

https://github.com/riscv-collab/riscv-gnu-toolchain#llvm--clang

Reply:
Thanks, I have removed the `--enable-linux` option in the PR.
https://github.com/riscv-collab/riscv-gnu-toolchain/pull/1276/files